src/hooks/useExtendedVoteData.js

```diff
@@ -76,3 +69,25 @@ export default function useExtendedVoteData(vote) {
     startTimestamp,
   }
 }
+
+export function useCanUserVote(vote) {
```

Comment:
Had to move this into its own hook since we also use it for the Decision card actions